crates/router/src/core/payout_link.rs

```diff
@@ -147,7 +148,7 @@ pub async fn initiate_payout_link(
             let enabled_payment_methods = link_data
                 .enabled_payment_methods
                 .unwrap_or(fallback_enabled_payout_methods.to_vec());
-
+            logger::debug!("enabled_payment_methods : {:?}", enabled_payment_methods);
```

Comment:
Does this serve any good purpose other than just logging? If not, we can remove it